### PR TITLE
test: speed-up it tests

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts [hunk 1]
```diff
@@ -40,6 +42,7 @@ const PACKAGE_MANAGERS_CMD: {[packageManager in SupportedPackageManagers]: {[com
   },
   yarn: {
     add: ['yarn', 'add'],
+    ci: ['yarn', 'install', '--frozen-lockfile'],
```
Comment (kpanot) [inline packages/@o3r/test-helpers/src/utilities/package-manager.ts:45]: Stupid question: `--frozen-lockfile` is only for yarn 1, is it the one we are using here?

**Review by kpanot — DISMISSED** [review]

File: packages/@o3r/test-helpers/src/utilities/package-manager.ts (hunk 1 above)
Comment (fpaul-1A) [reply packages/@o3r/test-helpers/src/utilities/package-manager.ts:45]: should be `--immutable` I guess, my bad

**Review by kpanot — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Comment by mrednic-1A** [comment]
Do we know how much do we gain, with approximation ?

**Comment by fpaul-1A** [comment]
> Do we know how much do we gain, with approximation ?

not very much I'm afraid, probably less than 5%

**Review by kpanot — APPROVED** [review]

**Pull request merged**